### PR TITLE
[ZZ-001] Update Site from Loot to Adventure Cards 

### PR DESCRIPTION
**Pull request by zzeleznick**
## Background

This PR enables a brave band of adventures, seeking adventure and rare loot, to build a community website.

Let creators and builders alike rejoice at this digital splendor.

For it was good (enough), and the hands and minds grow weary without a home on the web.

## Changes

- New site, same vibe as OG Loot project
- New assets and meta description
- New FAQ and resource page

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -35,32 +35,32 @@ function Head(): ReactElement {
...
-      <meta property="og:title" content="Loot" />
+      <meta property="og:url" content="https://0xadventures.com/" />
+      <meta property="og:title" content="Adventure Cards" />
       <meta
         property="og:description"
-        content="Loot is randomized adventurer gear generated and stored on chain."
+        content="Adventure Cards is a collection of randomized trading cards generated and stored on chain."
       />
-      <meta property="og:image" content="https://lootproject.com/meta.png" />
+      <meta property="og:image" content="https://0xadventures.com/meta.png" />
```
Comment (0xOlias) [inline components/Layout.tsx:53]: looks like this is a 404. not sure if the image we've been seeing (twitter/discord pfps) is available anywhere on 0xadventures.com yet.

File: pages/faq.tsx [hunk 2]
```diff
@@ -13,59 +13,45 @@ export default function FAQ(): ReactElement {
...
-            </a>
-            ).
-          </p>
-          <p>
-            Loot is the unfiltered, uncensorable building block for stories,
-            experiences, games, and more, in the hands of the community, at no
-            cost. Loot pursues complete decentralization from day one.
+            Adventure Cards is the first collectable card game stored entirely on chain.
+            Each NFT includes 45 procedurally generated cards, enough to play a full game.
+            Inspired by LOOT and MtG, Adventure cards are crypto primitives.
```
Comment (0xOlias) [inline pages/faq.tsx:46]: maybe spell out Magic: The Gathering for the kids in the back?

File: components/Layout.tsx (hunk 1 above)
Comment (typedarray) [inline components/Layout.tsx:53]: currently returns a 404, i was unable to find the image currently being used for pfp anywhere on 0xadventures.com. i added it to the deck viewer here https://deck-viewer.vercel.app/adv.jpeg

File: pages/faq.tsx (hunk 2 above)
Comment (typedarray) [inline pages/faq.tsx:46]: maybe spell out Magic: The Gathering for the kids in the back?

File: styles/pages/Home.module.scss [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
+    color: #edb900;
+    font-family: MedievalSharp, Georgia, serif;
   }
 
   p {
-    color: #8e8e8e;
-    font-size: 22px;
+    color: white;
+    text-align: left;
+    font-size: 32px;
```
Comment (typedarray) [inline styles/pages/Home.module.scss:17]: the 32 looks a bit big on my screen, perhaps max out at 24?

**Review by typedarray — APPROVED** [review]
lgtm 420 blaze it, couple small non-blocking nits

File: styles/pages/Home.module.scss (hunk 3 above)
Comment (zzeleznick) [reply styles/pages/Home.module.scss:17]: haha, i have really bad eyesight. I max out at 24px for smaller screen sizes. Open to toning it back a little.

File: pages/faq.tsx (hunk 2 above)
Comment (zzeleznick) [reply pages/faq.tsx:46]: same copytext as the OG https://0xadventures.com/ fwiw

File: components/Layout.tsx (hunk 1 above)
Comment (zzeleznick) [reply components/Layout.tsx:53]: Yeah, I created one myself too. https://github.com/Adventure-Cards/website/blob/zz/001/public/meta.png

Thought we would resolve this later once we build consensus on "official" org url.

File: pages/faq.tsx (hunk 2 above)
Comment (paulmars) [reply pages/faq.tsx:46]: it's wonderful reading this

**Comment by paulmars** [comment]
seems good to me, is it deployed to a "preview" url anywhere on vercel?

**Comment by paulmars** [comment]
looks great on the preview url

**Comment by 0xjjpa** [comment]
@zzeleznick Good stuff! Let's get it merged so we can started getting it previewed on Vercel.

@paulmars need to have a first commit since `gold` is the main branch and sadly Vercel doesn't get you to configure the branch on a first build.

**Comment by 0xjjpa** [comment]
Website available in https://website-opal-theta.vercel.app/, getting us a more specific domain

**Pull request merged**